ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -258,20 +258,7 @@ tests:
   as: vsphere-proxy-e2e-operator
   steps:
     cluster_profile: vsphere-2
-    post:
-    - chain: gather-network
-    - ref: gather-proxy
-    - chain: ipi-vsphere-post
-    - ref: vsphere-deprovision-bastionhost
-    pre:
-    - chain: ipi-conf-vsphere
-    - chain: vsphere-provision-bastionhost
-    - ref: ipi-conf-proxy
-    - ref: ovn-conf
-    - ref: ovn-conf-hybrid-manifest-with-custom-vxlan-port
-    - chain: ipi-install-vsphere
-    test:
-    - ref: windows-e2e-operator-test-with-custom-vxlan-port
```

Comment:
Not following here, is the test being removed?

Reply:
It should not be :facepalm: 

Thanks, I was wondering why pj-rehearse wasnt showing up